### PR TITLE
feat: APP-111 list project CTA

### PR DESCRIPTION
**Pull request by blushi**
## Description

https://regennetwork.atlassian.net/browse/APP-111

Needs https://github.com/regen-network/regen-server/pull/464 for list project + google login to work

Targeting `main` so we get have it deployed right away.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

From https://deploy-preview-2347--regen-marketplace.netlify.app/
1. Click "list a project" and log in via Keplr with an address that is not a class issuer, it should redirect you to the basic info form
2. Click "list a project" and log in via Keplr with an address that is a class issuer, it should redirect you to the choose a credit class form
3. Click "list a project" and log in via email, it should redirect you to the basic info form
4. **Please do not test this step before https://github.com/regen-network/regen-server/pull/464 is merged:** Click "list a project" and log in via google, it should redirect you to the basic info form
5. Logging in with the regular "log in" button should be unchanged

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie @clevinson see testing instructions

File: web-www/components/organisms/BackgroundImage/BackgroundImage.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { Box, BoxProps, SxProps } from '@mui/material';
```
Comment (blushi) [inline web-www/components/organisms/BackgroundImage/BackgroundImage.tsx:1]: in web-www: just some lint fixes

**Comment by erikalogie** [comment]
LGTM!

**Review by aaronc — APPROVED** [review]
utACK

**Comment by erikalogie** [comment]
Just noticed one small thing @blushi if we could fix the list a project CTA on mobile: https://monosnap.com/file/oQQkRWxFHVKeoBqkzWGFxDZtA8JzA7. You can refer to this comp in FIgma for ideal spacing and sizing: https://www.figma.com/file/MuSpCtCdU2ns4cFAsPfvsx/Text-Styles-%26-Components-(current)?type=design&node-id=3%3A192&mode=design&t=HH8rE4ZTJFMwHJkc-1

**Review by clevinson — APPROVED** [review]
Just tested all the flows above, and generally looks good to me.

@erikalogie Another alternative to the mobile issue is to have the mobile view within the hamburger menu dropdown if it's hard to squeeze it next to the login button. Happy to go with whatever you prefer, but I wanted to share this as a potential alternative to the design you shared.

Happy to have this merged once the mobile issue gets addressed one way or another.

**Comment by erikalogie** [comment]
@clevinson yeah I thought about that too but I was thinking at least for now it is nice to have that CTA more visible

**Pull request merged**